### ISSUE TITLE
Adding User Profile section

### ISSUE THREAD

**Issue by amrit03b** [amrit03b/TravelTales#15]
Allow users to create profiles where they can showcase their bio, profile picture, and a list of their published travel journals.

**Comment by seabeePraveen** [comment]
hello @amrit03b i am very interested in working with this issue if u please assign me this issue in the name of "hacktoberfest" i will just finish this in 24 hours 
Happy coding :)

**Comment by amrit03b** [comment]
> hello @amrit03b i am very interested in working with this issue if u please assign me this issue in the name of "hacktoberfest" i will just finish this in 24 hours Happy coding :)

Great! Assigned to you, go ahead.

**Comment by seabeePraveen** [comment]
@amrit03b as this is a frontend project do u want me to make only the frontend(HTML, CSS, JavaScript) is it? or what?
and also please confirm to the other issue also that i've been assigned for that also

**Comment by amrit03b** [comment]
> @amrit03b as this is a frontend project do u want me to make only the frontend(HTML, CSS, JavaScript) is it? or what? and also please confirm to the other issue also that i've been assigned for that also

I have created the Login/SignUp Page using firebase. After successful Login the user should be redirected to his Profile Section where they can showcase their bio, profile picture, and a list of their published travel journals. As of now you can proceed with the frontend part which will be opened when the user successfully logins, and you can use local storage for storing the data.

**Comment by seabeePraveen** [comment]
yeah okay then i will start working with the issue and i will close this issue by tommorow
Thank You!

**Comment by amrit03b** [comment]
> yeah okay then i will start working with the issue and i will close this issue by tommorow Thank You!

Great. Happy Coding :)

**Comment by amrit03b** [comment]
> yeah okay then i will start working with the issue and i will close this issue by tommorow Thank You!

Any updates?

**Comment by seabeePraveen** [comment]
yeah i've finished some part i will merge once the remaining part was done like i was also solving the other issues so i took some time

**Comment by amrit03b** [comment]
> yeah i've finished some part i will merge once the remaining part was done like i was also solving the other issues so i took some time

Ok, take your time.
